src/modules/events/events.js

```diff
@@ -29,3 +30,11 @@ bus.on('change', (change) => {
 		else winston.info('Events registered');
 	});
 });
+
+bus.on('http/init', app => {
+	app.use(route.post('/x/change', function *() {
```

Comment:
@satya164 I agree, It should be moved to some other module. Will do that but there is nothing wrong in allowing change events from this endpoint. You will have to send the sessionID as part of the the event for the change to be successful.